test/hocon_tests.erl

```diff
@@ -133,6 +133,7 @@ commas_test_() ->
     , ?_assertError(_, binary("a={b=1,c=2,,}"))
     , ?_assertError(_, binary("a={,b=1,c=2}"))
     , ?_assertError(_, binary("a={b=1,,c=2}"))
+    , ?_assertEqual(binary(""), binary("{}"))
```

Comment:
split to two `_assertEqual`s
?_assertEqual(#{}, binary("{}"))
?_assertEqual(#{}, binary(""))

Reply:
fixed